lib/pure/asyncdispatch.nim

```diff
@@ -444,7 +444,16 @@ when defined(windows) or defined(nimdoc):
 
     PCustomOverlapped* = ref CustomOverlapped
 
+    PCD = object
+      ioPort: Handle
+      handleFd: AsyncFD
+      waitFd: Handle
+      ovl: PCustomOverlapped
+    PPCD = ptr PCD
+
     AsyncFD* = distinct int
+
+    Callback = proc (fd: AsyncFD): bool {.closure,gcsafe.}
```

Comment:
But this type uses `AsyncFD` which declared only inside branches, and on windows this is:
`AsyncFD* = distinct int`
but on Posix systems
`AsyncFD* = distinct cint`